src/python/google_cloud_utils/big_query.py

```diff
@@ -379,8 +379,18 @@ def insert(self, inserts):
       time.sleep(1)
 
       if not result:
+        # Use result from the first batch, appending errors from the rest.
```

Comment:
Why we don't capture result (non-errors) from other parts of query response. This seems weird. @oliverchang for review as well.

Reply:
Agreed that there is nothing much else in the returned values.